barcode_detection/boundingbox/bounding_box.py

```diff
@@ -0,0 +1,6 @@
+class BoundingBox:
```

Comment:
I would rename `boundingbox` folder to smth more general, that refers to common data structures. Like `core` or `odm` (operational data model)